DEVELOPERS.md

```diff
@@ -51,79 +50,97 @@ three general steps:
 
 Having done so, typing `asdf` will yield a bunch of help text:
 
-    $ asdf
-    version: v0.11.2-8eb11b8
+```sh
+$ asdf
+version: v0.14.1-f00f759
 
-    MANAGE PLUGINS
-    asdf plugin add <name> [<git-url>]      Add a plugin from the plugin repo OR,
-    [...]
+MANAGE PLUGINS
+asdf plugin add <name> [<git-url>]      Add a plugin from the plugin repo OR,
+[...]
+```
 
-I like to have these four plugins (you can paste these commands):
+We like to have these four plugins (you can paste these commands):
 
-    asdf plugin add ruby
-    asdf plugin add direnv
-    asdf plugin add postgres
-    asdf plugin add nodejs
+```sh
+$ asdf plugin add ruby
+$ asdf plugin add direnv
+$ asdf plugin add postgres
+$ asdf plugin add nodejs
+```
 
 Once you have the plugins, you can start to install the software the
 plugin supports.  Let's first install Ruby.
 
 ### Installing Ruby
 
-First, install some system dependencies, e.g. a C and Rust compiler.
-[There is documentation listing of commands you can use for each
+First, install some system dependencies, such as a C and Rust compiler.
+[There is documentation listing the commands you can use for each
 platform](https://github.com/rbenv/ruby-build/wiki#suggested-build-environment)
 (e.g. Macintosh Homebrew, or Ubuntu).
 
 After that, install Ruby.  `asdf` will consult the `.tool-versions`
 file to select the version.
 
-    asdf install ruby
+```sh
+$ asdf install ruby
```

Comment:
Can you remove the `$` signs? When the copy button is used, it also copies the `$`, which invalidates the command.

<img width="1066" alt="Screenshot 2024-10-15 at 15 28 36" src="https://github.com/user-attachments/assets/8a45451a-bb53-4417-a21b-657c6c930106">

Reply:
Well, I agree that it makes copy-paste a bit harder, but I think this style makes it more clear what's a shell command vs. what's a Ruby code piece or what's just output, hence I added them consistently everywhere.

Let me pick other thoughts on this.

Reply:
Here's a Terrible Idea :tm:

```sh
#!/usr/bin/sh
asdf install ruby
```

Reply:
It's kinda wordy but maybe if we put it in the first time/once in a while it'll give people the idea.

Reply:
What do you think about using `#!/usr/bin/sh` for _some_ single lines and throw `$` but it for multi-lines with output?

Reply:
Since we couldn't get a strong signal, I will resolve this for now. @enescakir If you feel strong about it, I'm happy to change.